app/presenters/queries/available_translations.rb

```diff
@@ -25,7 +25,7 @@ def grouped_translations
 
       def expand_translation(id)
         expansion_rules = LinkExpansion::Rules
-        web_item(id).select { |f| expansion_rules.expansion_fields(:available_translations).include?(f) }
+        web_item(id).select { |f| expansion_rules.expansion_fields(:available_translations, nil).include?(f) }
```

Comment:
maybe this parameter could be optional or passed as a hash? The nil lacks any contextual meaning